src/osgEarthSymbology/TextSymbol.cpp

```diff
@@ -57,7 +57,8 @@ TextSymbol::TextSymbol(const TextSymbol& rhs,const osg::CopyOp& copyop):
                                                                            _minRange2ndlevel(rhs._minRange2ndlevel),
                                                                            _predefinedOrganisation(rhs._predefinedOrganisation),
                                                                            _predefinedOrganisationMargin( rhs._predefinedOrganisationMargin),
-                                                                           _backEarthCull(rhs._backEarthCull)
+                                                                           _backEarthCull(rhs._backEarthCull),
+                                                                           _textPriority(rhs._textPriority)
```

Comment:
**Impossible to understand the difference between TextSymbol::priority() and TextSymbol::textPriority**

replace textPriority by priorityFine everywhere